src/GafferScene/Options.cpp

```diff
@@ -102,7 +114,7 @@ IECore::ConstCompoundObjectPtr Options::computeProcessedGlobals( const Gaffer::C
 		IECore::DataPtr d = p->memberDataAndName( it->get(), name );
 		if( d )
 		{
-			result->members()["option:" + name] = d;
+			result->members()["option:" + prefixPlug()->getValue() + name] = d;
```

Comment:
Let's lift the `getValue()` call outside of the loop, to avoid repeating the same work. We might as well do the concatenation with "option:" outside too :

```
const std::string prefix = "option:" + prefixPlug()->getValue();
```